homeassistant/components/hue/light.py

```diff
@@ -213,12 +213,14 @@ def __init__(self, coordinator, bridge, is_group, light, supported_features):
             self.is_osram = False
             self.is_philips = False
             self.is_innr = False
+            self.is_livarno = False
             self.gamut_typ = GAMUT_TYPE_UNAVAILABLE
             self.gamut = None
         else:
             self.is_osram = light.manufacturername == "OSRAM"
             self.is_philips = light.manufacturername == "Philips"
             self.is_innr = light.manufacturername == "innr"
+            self.is_livarno = light.manufacturername == "_TZ3000_49qchf10"
```

Comment:
Perhaps it is better to change this line to:
> self.is_livarno = light.manufacturername.find("\_TZ3000\_") == 0

I have a led-strip from Lidl who resells livarno lights, and my light also needs this fix. The light has the same \_TZ3000\_ prefix but a different suffix.

Reply:
```suggestion
            self.is_livarno = light.manufacturername.startswith("_TZ3000_")
```